docs/basics/sso/setting-up-sso-special-cases.md

```diff
@@ -0,0 +1,34 @@
+---
+id: setting-up-sso-special-cases
+title: Setting Up SSO - Special Cases
+sidebar_label: Setting Up SSO - Special Cases
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<p><span className="sauceDBlue">Enterprise Plans only</span></p>
+
+## Single Identity Provider and Multiple Organizations at Sauce Labs
+
+:::note
+Integrating more than two Sauce Labs organizations with the same Identity Provider is not supported. If you have more than two organizations that you need integration with the same Identity Provider, contact your Customer Success Manager or [Sauce Labs Support](mailto:help@saucelabs.com).
+:::
+
+Some identity providers do not allow creating more than one SAML integration with the same service provider. In other words, they require one unique entity ID for a service provider.
+If your Identity Provider has this limitation and you have two organizations at Sauce Labs to integrate with SAML SSO for that Identity Provider, follow the setup steps below:
+
+1. In your Identity Provider set up the first SAML SSO integration/application in the [standard way](/basics/sso/setting-up-sso/#setting-up-identity-provider).
+2. Integrate the SAML SSO application](/basics/sso/setting-up-sso/#integrating-with-sauce-labs-service-provider) that you created in the previous step with one of your Sauce Labs organizations.
+3. In your Identity Provider set up another SAML SSO integration/application with the auxiliary Sauce Labs Service Provider.
+   - You have to use [different Sauce Labs metadata](https://accounts.saucelabs.com/am/sso/metadata/https%3A%2F%2Faccounts.saucelabs.com%2Fsp2).
+   - The metadata contains different entity ID and different ACS URLs (`sp2` instead of `sp`):
+   - | Setting                    | Value                                                                   |
```

Comment:
```suggestion
    | Setting                    | Value                                                                   |
```

Reply:
If I remove `-`, the table is not rendered. We have to keep this bullet point here.